src/app/find-help/material-support-search/material-support-search-form/material-support-search-form.component.html

```diff
@@ -0,0 +1,31 @@
+<form (ngSubmit)="search.emit(data)" #form="ngForm">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-4">
+        <label
+          ><b>{{ "TYPE_OF_SUPPORT" | translate }}</b></label
+        >
+        <mat-form-field appearance="outline">
+          <mat-select placeholder="Wybierz z listy" [(ngModel)]="categories" name="kategoriePomocy">
```

Comment:
`ngModel` needs to point to the selected value, not all of the values. This is likely what's causing the error as Angular tries to set the value to the whole collection and there's a mismatch.